frontend/src/screens/SearchResultsPage/models/EsModels.ts

```diff
@@ -0,0 +1,44 @@
+import { EsDataType } from '@screens/Search/models/EsDataTypes';
+import { ComplexityLevel } from '@models/LevelsEnum';
+
+// backend/core/src/main/java/com/knewless/core/elasticsearch/model/EsMapper.java
```

Comment:
required?

Reply:
This is a comment to look-up corresponding model on backend.